app/models/bulkrax/oai_entry.rb

```diff
@@ -37,7 +37,7 @@ def thumbnail_url
     def build_metadata
       self.parsed_metadata = {}
       self.parsed_metadata[work_identifier] = [record.header.identifier]
-      self.raw_metadata = { xml: record.metadata.to_s }
```

Comment:
the change to `record` here seems like it will cause anything that looks at raw_metadata to need to change, wouldn't it?

Reply:
I don't think anything looks at raw_metadata in the build process.  (Especially given that I just added the previous state recently.)